data/product.d/centos-stream.conf

```diff
@@ -0,0 +1,29 @@
+# Anaconda configuration file for CentOS Stream.
+
+[Product]
+product_name = CentOS Stream
+
+[Base Product]
+product_name = Red Hat Enterprise Linux
```

Comment:
Hi, it looks like the configuration file is just a copy-paste of `centos.conf`. In that case, you can change the `product_name` of the base product to `CentOS Linux` and remove the sections below. We will load them automatically from the base products.